src/Rules/Deprecations/AccessDeprecatedConstant.php

```diff
@@ -37,12 +37,71 @@ public function processNode(Node $node, Scope $scope): array
+            'UPDATE_REVOKED' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateManagerInterface::REVOKED instead.',
+            'UPDATE_NOT_SUPPORTED' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateManagerInterface::NOT_SUPPORTED instead.',
+            'UPDATE_NOT_CURRENT' => 'Deprecated in in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateManagerInterface::NOT_CURRENT instead.',
+            'UPDATE_CURRENT' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateManagerInterface::CURRENT instead.',
+            'UPDATE_NOT_CHECKED' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateFetcherInterface::NOT_CHECKED instead.',
+            'UPDATE_UNKNOWN' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateFetcherInterface::UNKNOWN instead.',
+            'UPDATE_NOT_FETCHED' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateFetcherInterface::NOT_FETCHED instead.',
+            'UPDATE_FETCH_PENDING' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\update\UpdateFetcherInterface::FETCH_PENDING instead.',
+            'USERNAME_MAX_LENGTH' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\user\UserInterface::USERNAME_MAX_LENGTH instead.',
+            'USER_REGISTER_ADMINISTRATORS_ONLY' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\user\UserInterface::REGISTER_ADMINISTRATORS_ONLY instead.',
+            'USER_REGISTER_VISITORS' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\user\UserInterface::REGISTER_VISITORS instead.',
+            'USER_REGISTER_VISITORS_ADMINISTRATIVE_APPROVAL' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\user\UserInterface::REGISTER_VISITORS_ADMINISTRATIVE_APPROVAL instead.',
+            // 'BlockInterface::BLOCK_LABEL_VISIBLE' => 'Deprecated in drupal:8.3.0 and is removed from drupal:9.0.0. Use \Drupal\Core\Block\BlockPluginInterface::BLOCK_LABEL_VISIBLE.',
+            // 'BlockInterface::BLOCK_REGION_NONE' => 'Deprecated in drupal:8.?.? and is removed from drupal:9.0.0.',
+            // 'WorkspaceInterface::DEFAULT_WORKSPACE' => 'Deprecated in drupal:8.8.0 and is removed from drupal:9.0.0. Use \Drupal\workspaces\WorkspaceManager::hasActiveWorkspace() instead.',
+            // 'PluralTranslatableMarkup::DELIMITER' => 'Deprecated in drupal:8.7.0 and is removed from drupal:9.0.0. Use Drupal\Component\Gettext\PoItem::DELIMITER instead.',
+            // 'FieldMigration::PLUGIN_METHOD' => 'Deprecated in drupal:8.7.0 and is removed from drupal:9.0.0. Use the migrate_drupal.field_discovery service instead. See https://www.drupal.org/node/3006076.',
```

Comment:
You know better than I why this is commented out, so I trust it, I'm more so curious -- why?

Reply:
These were the only non-global constants on api.drupal.org. I had a hunch that these would be found as there should be no PHP tokenizer problems associating the phpdoc to class constants.... I did not check though before I submitted this, so just left them there.

But indeed phpstan already finds these, proof of BlockInterface::BLOCK_REGION_NONE at https://dev.acquia.com/drupal9/deprecation_status/errors/271 :)

So we could just remove these and not commit them :)